js/repl/Repl.js

```diff
@@ -292,6 +304,10 @@ export default class Repl extends React.Component {
     };
   };
 
+  _compileToState = (code: string) => {
```

Comment:
Since we're already using this fancy property initializer syntax, may as well just call debounce here to avoid the extra code in the constructor:
```
_compileToState = debounce((code: string) => {
  this.setState(state => this._compile(code, state), this._persistState);
}, DEBOUNCE_DELAY);
```